.github/workflows/build_libraries.yml

```diff
@@ -0,0 +1,60 @@
+name: Build Libraries
+
+on:
+  push:
+    branches: [ develop, release/** ]
+    paths:
+      - 'Libraries/**'
+      - '.github/workflows/**'
+  pull_request:
+    branches: [ develop, release/** ]
+    paths:
+      - 'Libraries/**'
+      - '.github/workflows/**'
+
+env:
+    ROCM_VERSION: 6.2
+    AMDGPU_INSTALLER_VERSION: 6.2.60200-1
+
+jobs:
+    build:
+        name: "Build Libraries Examples"
+        runs-on: ubuntu-latest
+        container:
+            image: ubuntu:22.04
+        steps:
+          - uses: actions/checkout@v4
+
+          - name: Install dependencies
+            run: |
+                  apt-get update -qq &&
+                  apt-get install -y build-essential g++ glslang-tools \
+                    python3 python3-pip libglfw3-dev libvulkan-dev locales wget
+                  python3 -m pip install --upgrade pip
+                  python3 -m pip install cmake
+          - name: Install ROCm Dev
+            run: |
+                  export DEBIAN_FRONTEND=noninteractive
+                  wget https://repo.radeon.com/amdgpu-install/${{ env.ROCM_VERSION }}/ubuntu/jammy/amdgpu-install_${{ env.AMDGPU_INSTALLER_VERSION }}_all.deb
+                  apt-get -y install ./amdgpu-install_${{ env.AMDGPU_INSTALLER_VERSION }}_all.deb &&
+                  apt-get update -qq &&
+                  apt-get install -y \
```

Comment:
Installer the installer and then ignore it????

Reply:
Fix the runners then! Are they self hosted runners?

Reply:
Unfortunately, not. It is one of the basic runners.